### PR TITLE
Add Elyra image to OSC

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

**Comment by pacospace** [comment]
> can you fix the ci errors? :)

Done, thanks @HumairAK :)

File: kfdefs/overlays/osc/osc-cl1/jupyterhub/notebook-images/elyra-image.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  name: elyra-notebook-image
+  annotations:
+    opendatahub.io/notebook-image-url:
+      "https://github.com/thoth-station/s2i-lab-elyra"
+    opendatahub.io/notebook-image-name:
+      "Elyra Notebook Image"
+    opendatahub.io/notebook-image-desc:
+      "Jupyter notebook image with Elyra-AI installed."
+  labels:
+    app.kubernetes.io/instance: kfdefs-osc
```
[the remark this answers is not in the document]
Comment (pacospace) [reply kfdefs/overlays/osc/osc-cl1/jupyterhub/notebook-images/elyra-image.yaml]: Sure, done!

File: kfdefs/overlays/osc/osc-cl1/jupyterhub/notebook-images/elyra-image.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: elyra-notebook-image
+  annotations:
+    opendatahub.io/notebook-image-url:
+      "https://github.com/thoth-station/s2i-lab-elyra"
+    opendatahub.io/notebook-image-name:
+      "Elyra Notebook Image"
```
Comment (erikerlandson) [inline kfdefs/overlays/osc/osc-cl1/jupyterhub/notebook-images/elyra-image.yaml:10]: If this is custom image, should we annotate that in the `notebook-image-name` ?

Comment (pacospace) [reply kfdefs/overlays/osc/osc-cl1/jupyterhub/notebook-images/elyra-image.yaml:10]: What do you mean by custom image @erikerlandson? 

This is the odh official elyra image, there is also a custom-s2i-lab-elyra image, used to copy source content into the repo when building.

Comment (erikerlandson) [reply kfdefs/overlays/osc/osc-cl1/jupyterhub/notebook-images/elyra-image.yaml:10]: I thought we already had a notebook menu option that was labeled 'Elyra Notebook Image' but if we do not, then this is fine.

**Comment by HumairAK** [comment]
Image added

File: kfdefs/overlays/osc/osc-cl1/jupyterhub/notebook-images/elyra-image.yaml (hunk 2 above)
Comment (pacospace) [reply kfdefs/overlays/osc/osc-cl1/jupyterhub/notebook-images/elyra-image.yaml:10]: I see only two images linked to ArgoCD, is there some other image manually added?

**Pull request merged**